.github/workflows/ci.yml

```diff
@@ -23,10 +22,10 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install libkrb5-dev
-          pip install .[tests]
+          pip install -e .[tests]
       - name: Run tests
         run: |
-          pytest -s tests/ integration_tests/
+          pytest -s .
```

Comment:
Is there any way to verify that the tests running before this change also run after this change? i.e. none are skipped.

Reply:
I change it back